tests/transformers/tests/models/falcon/test_modeling_falcon.py

```diff
@@ -353,7 +353,7 @@ def test_past_key_values_format(self):
             outputs = model(**inputs)
 
             # If "past_key_values" is not returned, pass the test (e.g. RWKV uses a different cache name and format)
-            if "past_key_values" not in outputs or all(ele is None for ele in outputs["past_key_values"]):
```

Comment:
We should keep this change. It's not part of the source code so 100% sure it's not responsible for the regression.

Reply:
How about reverting only change in modeling_llama? I heard from @libinta that some tests were failing without this change but unfortunately I don't have more context.

Reply:
Sounds good to me! We'll patch Falcon later if there is an issue there too